blog/content/post/2022-04-07-fobnail_attestation_phase.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Fobnail Token - platform attestation
+abstract: 'The Fobnail Token is an open-source hardware USB device that helps to
+           determine the integrity of the system. The purpose of this blog post
+           is to present the development progress of this project. This phase
+           was focused on attestation.'
+cover: /covers/usb_token.png
+author:
+    - artur.kowalski
+    - krystian.hebel
+layout: post
+published: true
+date: 2022-04-06
+archives: "2022"
+
+tags:
+  - fobnail
+  - firmware
+  - tpm
+  - attestation
+categories:
+  - Firmware
+  - Security
+
+---
+
+# About the Fobnail Token project
+
+The Fobnail Token is a project that aims to provide a reference architecture for
+building offline integrity measurement servers on the USB device and clients
+running in Dynamically Launched Measured Environments (DLME). It allows the
+Fobnail owner to verify the trustworthiness of the running system before
+performing any sensitive operation. This project was founded by [NlNet
+Foundation](https://nlnet.nl/). More information about the project can be found
+in the [Fobnail documentation](https://fobnail.3mdeb.com/). Also, make sure to
+read other posts related to this project by visiting
+[fobnail](https://www.blog.3mdeb.com/tags/fobnail/) tag.
+
+# Scope of current phase
```

Comment:
I would appreciate if we would name the phases according to MoU and scope explained there. Can we do that?

Reply:
Well, we have an issue where name of phase doesn't match tasks inside that phase, and nobody I asked could answer why. I haven't even seen the MoU.